packages/toolpad-app/src/toolpad/ToolpadShell.tsx

```diff
@@ -13,14 +16,19 @@ import {
   Tooltip,
 } from '@mui/material';
 import HomeIcon from '@mui/icons-material/Home';
+import CloseIcon from '@mui/icons-material/Close';
 import HelpOutlinedIcon from '@mui/icons-material/HelpOutlined';
 import OpenInNewIcon from '@mui/icons-material/OpenInNew';
 import useMenu from '../utils/useMenu';
+import useLocalStorageState from '../utils/useLocalStorageState';
 
 const DOCUMENTATION_URL = 'https://mui.com/toolpad/getting-started/setup/';
```

Comment:
do we have specific instructions on how to update besides the setup instructions? should we?

Reply:
We could create a dialog within the app containing the instructions instead of redirecting?

Reply:
I think we can start with a link to the documentation. There's not much to be done from within the app to update Toolpad.